v2.0/admin-ui-access-and-navigate.md

```diff
@@ -107,6 +84,58 @@ Down Since | How long the node has been down.
 
 When you decommission a node, CockroachDB lets the node finish in-flight requests, rejects any new requests, and transfers all range replicas and range leases off the node so that it can be safely shut down. See [Remove Nodes](remove-nodes.html) for more information.
 
+### Node Map (Enterprise)
+
+The **Node Map** is an [enterprise-only](enterprise-licensing.html) feature that gives you a visual representation of the geographical configuration of your cluster. To set up and enable the Node Map, see [Enable Node Map](enable-node-map.html). The Node Map consists of the following components:
+
+**Node component**
+
+<img src="{{ 'images/v2.0/admin-ui-node-components.png' | relative_url }}" alt="CockroachDB Admin UI Summary Panel" style="border:1px solid #eee;max-width:90%" />
+
+**Region component**
+
+<img src="{{ 'images/v2.0/admin-ui-region-component.png' | relative_url }}" alt="CockroachDB Admin UI Summary Panel" style="border:1px solid #eee;max-width:90%" />
+
+### Time Series Graphs
```

Comment:
For this, the Summary panel, and the Events panel, don't we need to tell users how to get to these areas now that they're not available from the main ui landing page?

Reply:
Hmmm...yes..for the Time Series as well. Adding the navigation now.